src/libs/ReportUtils.js

```diff
@@ -702,8 +702,7 @@ function buildOptimisticIOUReport(ownerEmail, userEmail, total, chatReportID, cu
  */
 function getIOUReportActionMessage(type, total, participants, comment, currency) {
     const amount = NumberFormatUtils.format(preferredLocale, total / 100, {style: 'currency', currency});
-    const isMultipleParticipantReport = participants.length > 1;
-    const displayNames = _.map(participants, participant => getDisplayNameForParticipant(participant.login, isMultipleParticipantReport) || participant.login);
+    const displayNames = _.map(participants, participant => getDisplayNameForParticipant(allPersonalDetails[participant.login], true) || participant.login);
```

Comment:
This is throwing `text.replace` is not a function, I think due to the change [here](https://github.com/Expensify/App/pull/12399), `getDisplayNameForParticipant ` now expects a string instead of the `participant` object.